Userland/Applications/PixelPaint/Layer.cpp

```diff
@@ -180,6 +180,31 @@ void Layer::crop(Gfx::IntRect const& rect)
     did_modify_bitmap();
 }
 
+void Layer::resize(Gfx::IntSize const& new_size, Gfx::Painter::ScalingMode scaling_mode)
+{
+    const Gfx::IntRect old_rect(Gfx::IntPoint(0, 0), size());
```

Comment:
I believe the constness doesn't really matter here since it's a local variable anyway.
If you want to keep it, east coast pls ;)
I'd write it as `auto rect = Gfx::IntRect(foo, bar)` , but that is just my preference for initializing variables... I don't know if there's a style guideline for that.